src/Http/ResponseBuilder.php

```diff
@@ -54,15 +69,19 @@ public function __construct($responseFactory)
      *
      * @param  int|null $statusCode
      * @param  array    $headers
+     * @param  bool     $successFlag
      * @return \Illuminate\Http\JsonResponse
      */
-    public function respond(int $statusCode = null, array $headers = []):JsonResponse
+    public function respond(int $statusCode = null, array $headers = [], bool $successFlag = true):JsonResponse
```

Comment:
I don't feel like the flag belongs in the parameter list here, seeing as you can set it using the setter below.